itest/lnd_open_channel_test.go

```diff
@@ -279,20 +310,32 @@ func testOpenChannelUpdateFeePolicy(ht *lntest.HarnessTest) {
 		assertNodesPolicyUpdate(ht, nodes, bob, bobPolicy, chanPoint)
 
 		// They should now know about the default policies.
-		for _, node := range nodes {
+		for _, n := range nodes {
 			ht.AssertChannelPolicy(
-				node, alice.PubKeyStr, alicePolicy, chanPoint,
+				n, alice.PubKeyStr, alicePolicy, chanPoint,
 			)
 			ht.AssertChannelPolicy(
-				node, bob.PubKeyStr, bobPolicy, chanPoint,
+				n, bob.PubKeyStr, bobPolicy, chanPoint,
 			)
 		}
+
+		// We should be able to forward a payment from Carol to Bob
+		// through the new channel we opened.
+		payReqs, _, _ := ht.CreatePayReqs(bob, paymentAmt, 1)
+		ht.CompletePaymentRequests(carol, payReqs)
 	}
 
 	for i, feeScenario := range feeScenarios {
 		ht.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
 			st := ht.Subtest(t)
-			ht.EnsureConnected(alice, bob)
+			st.EnsureConnected(alice, bob)
+
+			st.RestartNode(carol)
```

Comment:
Is this needed because Carol is shut down after each test?

Reply:
This is a consequence of how the new itest framework works. When you have a long-running node and are using `ht.Subtest()`, you need to restart those nodes in order for their context to be refreshed. I added a comment.